app/Http/Controllers/ProjectController.php

```diff
@@ -16,7 +16,7 @@ public function __construct() {
 
 	public function index() {
 		$projects = Project::all();
-		return view('projects', compact('projects'));
+		return view('projects', compact('projects'), 'tags' => Tag::all());
```

Comment:
Mmm not sure that this works. Creo que el 3er argumento de view está esperando un status para la respuesta.

Si quieres pasarlo sería `return view('projects', ['projects' => $projects, 'tags' => Tag::all()])`
O ponlo arriba del statement `$tags = Tag::all()` y luego `compact('tags', 'projects')`

Reply:
done! el issue de code climate es el mismo de "constant instead of duplicating this literal"